src/checks/index.ts

```diff
@@ -32,6 +32,11 @@ const linters = async () => {
             url: 'https://github.com/datenanfragen/data/blob/master/src/checks/pack',
             path_filter: (path: string) => path.startsWith('company-packs/'),
         },
+        genericlint: {
+            checks: await importChecks('generic'),
+            url: 'https://github.com/datenanfragen/data/blob/master/src/checks/generic',
+            path_filter: (path: string) => path.startsWith('companies/') || path.startsWith('company-packs/') || path.startsWith('supervisory-authorities/') || path.startsWith('suggested-companies/'),
+        },
```

Comment:
Do we need to establish another check type for this? The suggested companies are deprecated (and can actually be removed from the repo now). If we made this a `record` check, we would indeed miss the company packs, but I feel like this is less critical for them (a missing company pack is a lot more obvious than a missing company). What do you think?

Reply:
Actually: If we decide, we only need this for `record`s, do we still need the additional check at all? I think `match-filename-slug` should cover this already in that case.